### DISCUSSION TITLE
ISV Gurobi License Key + or-tools?

### DISCUSSION THREAD

**Discussion by ruubelly** [google/or-tools#2502] [Build and configuration issues] [answered]
Hello, I'm unsure if this has been definitively answered elsewhere. If I have an ISV Gurobi License, am I able to use this license with OR Tools?

From reading #1625 (now almost 2 years old) and taking a small peek at the code ([gurobi_interface.cc](https://github.com/google/or-tools/blob/fa84bc05e72641dddfbb98164d81b8bc9bef6ea5/ortools/linear_solver/gurobi_interface.cc#L1320)), it seems like the or tools implementation is meant for floating use license but **not** the ISV license. Could someone confirm? Thanks in advance!

**Comment by lperron** [comment, marked as answer]
yes, it is not included.

You will need to build from sources, after having patched
gurobi/environment.cc (on the master branch).
I will see if we can include the code in an upcoming release.

Thanks
Laurent Perron | Operations Research | ***@***.*** | (33) 1 42 68 53
00
